include/sharpSAT/stopwatch.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * stopwatch.h
+ *
+ *  Created on: Jun 12, 2018
+ *      Author: Radomir Cernoch
+ */
+
+#ifndef SHARP_SAT_STOPWATCH_H_
+#define SHARP_SAT_STOPWATCH_H_
+
+#include <sys/time.h>
```

Comment:
I think it would be better to implement this class in terms of `<chrono>`, to simplify later potential porting to systems that are not Linux.

master-keying/mks should have something similar in the branch `dev-solver-timings`, split across `src/main/cpp/stopwatch.hpp` and `src/main/cpp/stopwatch.cpp`.